pyproject.toml

```diff
@@ -27,6 +27,7 @@ uvicorn = "=0.10.0"
 django = {version = "^2.0",optional = true}
 graphql-core = {version = "^3.0.0a0", allows-prereleases = true}
 aniso8601 = "^8.0.0"
+flask = "^1.1"
```

Comment:
can you add this to the extras? Like we've done for django? 😊

Reply:
Sure will do